docs/Servers.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Kettle Servers
+layout: default
+category: Kettle
+---
+
+Kettle includes two builtin grades, [kettle.server](#kettle.server) for defining a plain HTTP server, and a derived grade [kettle.server.ws)(#kettle.server.ws) for defining a server
```

Comment:
Broken link here thanks to the trailing parenthesis mating with the opening square bracket.